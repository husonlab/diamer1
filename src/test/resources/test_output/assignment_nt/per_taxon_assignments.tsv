node id	rank	label	kmers in database	kmer count	kmer count (accumulated)	OVO (0.20) read count	OVO (0.20) read count cumulative	OVO (0.20) read count (norm. kmers)	OVO (0.20) read count (norm. kmers) cumulative	OVO (0.50) read count	OVO (0.50) read count cumulative	OVO (0.50) read count (norm. kmers)	OVO (0.50) read count (norm. kmers) cumulative	OVO (0.60) read count	OVO (0.60) read count cumulative	OVO (0.60) read count (norm. kmers)	OVO (0.60) read count (norm. kmers) cumulative	OVO (0.80) read count	OVO (0.80) read count cumulative	OVO (0.80) read count (norm. kmers)	OVO (0.80) read count (norm. kmers) cumulative	OVO (0.90) read count	OVO (0.90) read count cumulative	OVO (0.90) read count (norm. kmers)	OVO (0.90) read count (norm. kmers) cumulative	OVO (1.00) read count	OVO (1.00) read count cumulative	OVO (1.00) read count (norm. kmers)	OVO (1.00) read count (norm. kmers) cumulative	OVA (0.70) read count	OVA (0.70) read count cumulative	OVA (0.70) read count (norm. kmers)	OVA (0.70) read count (norm. kmers) cumulative	normalized kmer count
0	no rank	root	0	0	79	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0	2	0.0
1	1 from root	rank1	113	11	11	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.09734513274336283
2	1 from root	rank2	237	68	68	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.2869198312236287
4	2 from root	organism2	334	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
5	2 from root	organism3	318	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
3	2 from root	organism1	82	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
