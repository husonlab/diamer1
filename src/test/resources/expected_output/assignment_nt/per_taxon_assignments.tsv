node id	rank	label	kmers in database	kmer count	kmer count (accumulated)	OVO (0.20) read count	OVO (0.20) read count (norm. kmers)	OVO (0.20) cumulative read count	OVO (0.20) cumulative read count (norm. kmers)	OVO (0.50) read count	OVO (0.50) read count (norm. kmers)	OVO (0.50) cumulative read count	OVO (0.50) cumulative read count (norm. kmers)	OVO (0.60) read count	OVO (0.60) read count (norm. kmers)	OVO (0.60) cumulative read count	OVO (0.60) cumulative read count (norm. kmers)	OVO (0.80) read count	OVO (0.80) read count (norm. kmers)	OVO (0.80) cumulative read count	OVO (0.80) cumulative read count (norm. kmers)	OVO (0.90) read count	OVO (0.90) read count (norm. kmers)	OVO (0.90) cumulative read count	OVO (0.90) cumulative read count (norm. kmers)	OVO (1.00) read count	OVO (1.00) read count (norm. kmers)	OVO (1.00) cumulative read count	OVO (1.00) cumulative read count (norm. kmers)	OVA (0.70) read count	OVA (0.70) read count (norm. kmers)	OVA (0.70) cumulative read count	OVA (0.70) cumulative read count (norm. kmers)	normalized kmer count
0	no rank	root	0	0	78	0	0	2	2	0	0	2	2	0	0	2	2	0	0	2	2	0	0	2	2	0	0	2	2	0	0	2	2	0.0
1	1 from root	rank1	113	10	10	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.08849557522123894
2	1 from root	rank2	237	68	68	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.2869198312236287
4	2 from root	organism2	336	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
5	2 from root	organism3	318	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
3	2 from root	organism1	82	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
